src/rdiff-backup-delete

```diff
@@ -60,6 +64,71 @@ def _str(value):
         return value.decode('utf-8', errors='replace')
 
 
+# Check if zcat is available.
+GZIP = spawn.find_executable('gzip')
+
+
+class _wrap_close:
+    """
+    Helper for _open() -- a proxy for a file whose close waits for the process.
+    """
+
+    def __init__(self, stream, proc):
+        self._stream = stream
+        self._proc = proc
+
+    def close(self):
+        if self._proc.stdin:
+            self._proc.stdin.close()
+        returncode = self._proc.wait()
+        if self._proc.stdout:
+            self._proc.stdout.close()
+        if returncode == 0:
+            return None
```

Comment:
Why doing this? Why not returning directly the returncode? It's mostly a moot point as I didn't see that you do anything with it, but IMHO having consistently an integer being returned is easier to handle.